api/routes/api_news.py

```diff
@@ -39,11 +40,26 @@ def get_sport_news_endpoint(sport_type):
 @cache.cached(timeout=60*3)
 def get_popular_news_endpoint():
     try:
-        recent_news = get_popular_news(5, session)
-        return get_good_response(recent_news)
+        pouplar_news = get_popular_news(5, session)
+        return pouplar_news
     except Exception as e:
         response = {"error in service": str(e)}
         return get_error_response(response, 500)
+    
+@news_app.route('/article', methods=['POST'])
+def specific_article():
+
+    try:
+        article = request.get_json()
+        news_id=article['blob_id']
+        response = get_news_by_id(news_id, session)
+        return response
+    except Exception as e:
+        response = {"error in service":  str(e)}
+        return get_error_response(response, 500)
```

Comment:
WE have a whole bunch of places where we have to manually hardcode status code to be 500. I suggest creating separate "something" for handling unknown exceptions and just use it instead.